src/eva/vision/data/datasets/segmentation/total_segmentator.py

```diff
@@ -18,14 +18,11 @@
 class TotalSegmentator2D(base.ImageSegmentation):
     """TotalSegmentator 2D segmentation dataset."""
 
-    _train_index_ranges: List[Tuple[int, int]] = [(0, 83)]
-    """Train range indices."""
-
-    _val_index_ranges: List[Tuple[int, int]] = [(83, 103)]
-    """Validation range indices."""
-
-    _n_slices_per_image: int = 20
```

Comment:
do we remove the option to sub-sample slices?

Reply:
yes removed as not sure if we need it and it complicates the dataset a bit; do you think we will need it?

Reply:
i think it would at least be interesting for performance-experiments.. as slices next too each other are very similar i would expect to get almost as good results with much less data.

Reply:
cool I added it back then